### PR TITLE
PROC-1296 List payment/settlement captures documentation

### PR DESCRIPTION
**Pull request by Smitsel**
@willemstuursma you've mentioned an update to the changelog in your API endpoint PR.
Let me know what needs to be added :)

### CODE REVIEW

**Comment by willemstuursma** [comment]
I've added the changes @Smitsel, can you review?

**Comment by Smitsel** [comment]
I'll hand it over to @mvdpanne. Can't approve my own PR. Your changes LGTM.

File: source/reference/v2/captures-api/get-capture.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ Get Payment Capture
 
 Retrieve a single capture by its ID. Note the original payment's ID is needed as well.
 
-Captures are used for in payments that have the *authorize-then-capture* flow. The only payment methods at the moment
+Captures are used in payments that have the *authorize-then-capture* flow. The only payment methods at the moment
```
[the remark this answers is not in the document]
Comment (Smitsel) [reply source/reference/v2/captures-api/get-capture.rst]: √

File: source/reference/v2/captures-api/list-captures.rst [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+   :method: GET
+   :url: https://api.mollie.com/v2/payments/*paymentId*/captures
+
+.. authentication::
+   :api_keys: true
+   :oauth: true
+
+Retrieve all captures for a certain payment.
+
+Captures are used in payments that have the *authorize-then-capture* flow. The only payment methods at the moment
```
[the remark this answers is not in the document]
Comment (Smitsel) [reply source/reference/v2/captures-api/list-captures.rst]: √

File: source/reference/v2/captures-api/list-captures.rst [hunk 3]
```diff
@@ -0,0 +1,183 @@
+List Captures
```
[the remark this answers is not in the document]
Comment (Smitsel) [reply source/reference/v2/captures-api/list-captures.rst]: √

File: source/reference/v2/captures-api/list-captures.rst [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+                           "type": "text/html"
+                       }
+                   }
+               }
+           ]
+       },
+       "count": 1,
+       "_links": {
+           "documentation": {
+               "href": "https://docs.mollie.com/reference/v2/captures-api/list-capture",
```
[the remark this answers is not in the document]
Comment (Smitsel) [reply source/reference/v2/captures-api/list-captures.rst]: √

File: source/reference/v2/settlements-api/list-settlement-captures.rst [hunk 5]
```diff
@@ -0,0 +1,168 @@
+List Settlement Captures
```
[the remark this answers is not in the document]
Comment (Smitsel) [reply source/reference/v2/settlements-api/list-settlement-captures.rst]: √

File: source/reference/v2/settlements-api/list-settlement-captures.rst [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+   :method: GET
+   :url: https://api.mollie.com/v2/settlements/*settlementId*/captures
+
+.. authentication::
+   :api_keys: true
+   :oauth: true
+
+Retrieve all captures for a certain settlement.
+
+Captures are used in payments that have the *authorize-then-capture* flow. The only payment methods at the moment
```
[the remark this answers is not in the document]
Comment (Smitsel) [reply source/reference/v2/settlements-api/list-settlement-captures.rst]: √

File: source/reference/v2/captures-api/get-capture.rst [hunk 7]
```diff
@@ -58,7 +58,7 @@ Response
 
        .. type:: string
 
-     - The capture's unique identifier, for example ``cpt_4qqhO89gsT``.
+     - The order's unique identifier, for example ``cpt_4qqhO89gsT``.
```
Comment (mvdpanne) [inline source/reference/v2/captures-api/get-capture.rst]: I think this should stay `The capture's unique identifier`?

Comment (Smitsel) [reply source/reference/v2/captures-api/get-capture.rst]: Yes, don't know how this happened

**Comment by Smitsel** [comment]
When the settlement change is done to the API this one can get merged @willemstuursma

**Comment by willemstuursma** [comment]
No, only if it has them. I'll update the docs.

**Comment by sandervanhooft** [comment]
FYI I'm using this now in the tests:

```json
           "_links":{
              "self":{
                 "href":"https://api.mollie.com/v2/payments/tr_WDqYK6vllg",
                 "type":"application/hal+json"
              },
              "documentation":{
                 "href":"https://docs.mollie.com/reference/v2/payments-api/get-payment",
                 "type":"text/html"
              },
              "captures":{
                "href":"https://api.mollie.com/v2/payments/tr_WDqYK6vllg/captures",
                "type":"application/hal+json"
              }
           }
```

**Comment by sandervanhooft** [comment]
And just to be sure: a capture has no status?

**Comment by willemstuursma** [comment]
> And just to be sure: a capture has no status?

It is, or it is not. There's no status.

**Pull request merged**